test/unix.js

```diff
@@ -38,7 +38,7 @@ if (process.platform !== 'win32') {
           bufs.push(buf)
         })
         body.on('end', () => {
-          t.strictEqual('hello', Buffer.concat(bufs).toString('utf8'))
+          t.strictEqual(Buffer.concat(bufs).toString('utf8'), 'hello')
```

Comment:
Why?

Reply:
I was confusing `expected` and `actual` in `tap` output. I forgot to change it back.